ludwig/api.py

```diff
@@ -65,6 +65,10 @@
 # logger.setLevel(logging.INFO)
 
 
+logger = logging.getLogger(__name__)
```

Comment:
What is this change relevant to, is this a bug fix?

Reply:
i guess it's a byproduct of merging from master, maybe merging again from master to be sure about these logger things would be a good idea

Reply:
Reverted the state to the one at Uber  master branch.

```suggestion
logger = logging.getLogger(__name__)


# logger.setLevel(logging.INFO)
```